<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsPageWidget</class>
 <widget class="QWidget" name="SettingsPageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>605</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="topHorizontalLayout">
     <item>
      <widget class="QListWidget" name="listWidget">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>16777215</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>SNMP</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Устройства</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="snmpPage">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QGroupBox" name="snmpSettingsGroupBox">
           <property name="title">
            <string>Параметры SNMP</string>
           </property>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="readComLabel">
              <property name="text">
               <string>Read community</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="readComEdit"/>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="writeComEdit"/>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="timeoutEdit"/>
            </item>
            <item row="5" column="1">
             <widget class="QLineEdit" name="portEdit"/>
            </item>
            <item row="6" column="1">
             <widget class="QLineEdit" name="retriesEdit"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="writeComLabel">
              <property name="text">
               <string>Write community</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="timeoutLabel">
              <property name="text">
               <string>Таймаут, мс</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="portLabel">
              <property name="text">
               <string>Порт</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="retriesLabel">
              <property name="text">
               <string>Повторы</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="saveTimeoutLabel">
              <property name="text">
               <string>Таймаут сохранения, мс</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="saveTimeoutEdit"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="devicePage">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QTabWidget" name="deviceTabWidget">
           <property name="currentIndex">
            <number>1</number>
           </property>
           <widget class="QWidget" name="switchTab">
            <attribute name="title">
             <string>Switch</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QGroupBox" name="swServiceVlanGroupBox">
               <property name="title">
                <string>Вланы для сервисов</string>
               </property>
               <layout class="QFormLayout" name="formLayout_2">
                <item row="0" column="0">
                 <widget class="QLabel" name="swInternetVlanLabel">
                  <property name="text">
                   <string>Интернет</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="swIptvUnicastVlanLabel">
                  <property name="text">
                   <string>IPTV Unicast</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLineEdit" name="swInternetVlanEdit"/>
                </item>
                <item row="1" column="1">
                 <widget class="QLineEdit" name="swIptvUnicastVlanEdit"/>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="dslamTab">
            <attribute name="title">
             <string>Dslam</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QGroupBox" name="dsDslProfileGroupBox">
               <property name="title">
                <string>Dsl профили</string>
               </property>
               <layout class="QGridLayout" name="gridLayout">
                <item row="0" column="1" colspan="2">
                 <widget class="QComboBox" name="dsDeviceModelComboBox">
                  <item>
                   <property name="text">
                    <string>MA5600</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>MA5300</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>MXA-32</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>MXA-64</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="3" column="0" colspan="6">
                 <widget class="QTableView" name="dsDslProfileTableView">
                  <property name="contextMenuPolicy">
                   <enum>Qt::ActionsContextMenu</enum>
                  </property>
                  <property name="editTriggers">
                   <set>QAbstractItemView::EditKeyPressed</set>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="dsDeviceLabel">
                  <property name="text">
                   <string>Модель:</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0" colspan="6">
                 <layout class="QHBoxLayout" name="dsEditProfileButtonsHLayout">
                  <item>
                   <widget class="QToolButton" name="dsAddProfileButton">
                    <property name="toolTip">
                     <string>Добавить новый DSL профиль...</string>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                    <property name="icon">
                     <iconset resource="../resource.qrc">
                      <normaloff>:/images/add.png</normaloff>:/images/add.png</iconset>
                    </property>
                    <property name="iconSize">
                     <size>
                      <width>22</width>
                      <height>22</height>
                     </size>
                    </property>
                    <property name="autoRaise">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QToolButton" name="dsEditProfileButton">
                    <property name="toolTip">
                     <string>Отредактировать информацию о Dsl профиле...</string>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                    <property name="icon">
                     <iconset resource="../resource.qrc">
                      <normaloff>:/images/edit.png</normaloff>:/images/edit.png</iconset>
                    </property>
                    <property name="iconSize">
                     <size>
                      <width>22</width>
                      <height>22</height>
                     </size>
                    </property>
                    <property name="autoRaise">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QToolButton" name="dsRemoveProfileButton">
                    <property name="toolTip">
                     <string>Удалить существующий DSL профиль...</string>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                    <property name="icon">
                     <iconset resource="../resource.qrc">
                      <normaloff>:/images/remove.png</normaloff>:/images/remove.png</iconset>
                    </property>
                    <property name="iconSize">
                     <size>
                      <width>22</width>
                      <height>22</height>
                     </size>
                    </property>
                    <property name="autoRaise">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_2">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
                <item row="0" column="3">
                 <widget class="QLabel" name="dsTypeBoardLabel">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>Тип платы:</string>
                  </property>
                  <property name="scaledContents">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item row="0" column="4" colspan="2">
                 <widget class="QComboBox" name="dsTypeBoardComboBox">
                  <item>
                   <property name="text">
                    <string>adsl</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>shdsl</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="viewPage">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QGroupBox" name="styleGroupBox">
           <property name="title">
            <string>Стиль</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="viewSettingsGroupBox">
           <property name="title">
            <string>Параметры отображения таблиц</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="bottomHorizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="defaultSettingButton">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>38</height>
        </size>
       </property>
       <property name="text">
        <string>По  умолчанию</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveSettingsButton">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>38</height>
        </size>
       </property>
       <property name="text">
        <string>Сохранить</string>
       </property>
       <property name="icon">
        <iconset resource="../resource.qrc">
         <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="addDslProfileAction">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/images/add.png</normaloff>
     <normalon>:/images/add.png</normalon>:/images/add.png</iconset>
   </property>
   <property name="text">
    <string>Добавить новый профиль</string>
   </property>
   <property name="toolTip">
    <string>Добавить новый профиль</string>
   </property>
  </action>
  <action name="editDslProfileAction">
   <property name="icon">
    <iconset>
     <normalon>:/images/edit.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Изменить текущую ячейку</string>
   </property>
   <property name="toolTip">
    <string>Изменить текущую ячейку</string>
   </property>
  </action>
  <action name="removeDslProfileAction">
   <property name="icon">
    <iconset>
     <normalon>:/images/remove.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Удалить текущий профиль</string>
   </property>
   <property name="toolTip">
    <string>Удалить текущий профиль</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>listWidget</tabstop>
  <tabstop>readComEdit</tabstop>
  <tabstop>writeComEdit</tabstop>
  <tabstop>timeoutEdit</tabstop>
  <tabstop>saveTimeoutEdit</tabstop>
  <tabstop>portEdit</tabstop>
  <tabstop>retriesEdit</tabstop>
  <tabstop>deviceTabWidget</tabstop>
  <tabstop>swInternetVlanEdit</tabstop>
  <tabstop>swIptvUnicastVlanEdit</tabstop>
  <tabstop>dsDeviceModelComboBox</tabstop>
  <tabstop>dsTypeBoardComboBox</tabstop>
  <tabstop>dsAddProfileButton</tabstop>
  <tabstop>dsEditProfileButton</tabstop>
  <tabstop>dsRemoveProfileButton</tabstop>
  <tabstop>dsDslProfileTableView</tabstop>
  <tabstop>defaultSettingButton</tabstop>
  <tabstop>saveSettingsButton</tabstop>
 </tabstops>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
